linter_exclusions.yml

```diff
@@ -540,6 +540,11 @@ dataprotection backup-instance restore initialize-for-data-recovery-as-files:
     target_blob_container_url:
       rule_exclusions:
       - option_length_too_long
+dataprotection backup-instance update-msi-permissions:
+  parameters:
+    target_storage_account_id:
+      rule_exclusions:
+      - option_length_too_long
```

Comment:
Can you add a short option for this parameter instead of adding exclusion rule?

Reply:
We can't think of any meaningful alternative name that's also user-friendly. Avoiding `target-sa-id` as it isn't clear what sa is and `target-storage-id` as we need to specify storage account.